packages/botfuel-dialog/src/adapters/shell-adapter.js

```diff
@@ -50,10 +52,47 @@ class ShellAdapter extends Adapter {
     });
   }
 
+  /* eslint-disable no-console */
   /** @inheritDoc */
   async sendMessage(botMessage) {
-    // eslint-disable-next-line no-console
-    console.log(chalk.hex('#16a085')(`${DELIMITER}${botMessage.payload.value}`));
+    if (botMessage.type === 'actions') {
+      await this.sendActionsMessage(botMessage);
+    } else {
+      console.log(chalk.hex(this.botTextColor)(`${DELIMITER}${botMessage.payload.value}`));
+    }
+  }
+
+  /**
+   * Send actions message
+   * @param {Object} actionsMessage actionMessage
+   * @returns {Promise.<void>}
+   */
+  async sendActionsMessage(actionsMessage) {
+    const actions = actionsMessage.payload.value;
+    let str = 'Please select:\n';
+    actions.forEach((action, index) => {
+      str += `${index + 1}: (${action.type}) ${action.text}\n`;
+    });
+
```

Comment:
"Pure" way of doing this (not really important, can be useful if you decide to put this in a function outside `sendActionsMessage`):

```javascript
const choices = actions
    .map((action, index) => `${ index + 1 }: (${action.type}) ${action.text}\n`);

const question = `Please select:\n${choices}'
```